contrib/zenazn/goji.v1/web/option.go

```diff
@@ -30,7 +33,11 @@ func defaults(cfg *config) {
 	} else {
 		cfg.analyticsRate = globalconfig.AnalyticsRate()
 	}
-	cfg.serviceName = "http.router"
+	cfg.serviceName = namingschema.NewServiceNameSchema(
```

Comment:
If I understand this correctly isn't this a change in default behavior?
A previous user of `goji.v1` who has a global service name configured would before this change get service name `http.router` but now will get the globally configured service name since that takes priority over the default service name?

Reply:
in line 36, I add the option:
```
namingschema.WithVersionOverride(namingschema.SchemaV0, defaultServiceName),
```
which basically what it does is to always set `http.router` for V0, which is current behavior. So what you describe will only happen for v1.

Let me know if it's more clear now!